mkosi.conf.d/20-serpentos.conf

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+ShimBootloader=none
+Packages=
```

Comment:
Instead of overriding the package list we should move the packages that aren't available in SerpentOS to the distribution specific configuration files.

Also, what about `less`, I'd be very surprised if that isn't packaged for SerpentOS already.